src/hooks/usePlaceOrder.ts

```diff
@@ -47,23 +47,41 @@ export const usePlaceOrder = (): Result => {
           exchangeApi.getCurrentBatchId(),
         ])
 
-        const validUntil = batchId + DEFAULT_ORDER_DURATION
+        if (sellTokenId !== 0 || buyTokenId !== 0) {
```

Comment:
would this `0` be worth making into a const like `OWL_TOKEN_ID` or is it always zero?

Reply:
also a comment note would be useful coming back to it later we'll likely forget.

Reply:
totaly, good point.

I this case, this 0 means, that is not a valid token. I need to review a bit why the contract is returning 0 for tokens that don't know, apparently it should be reverting according to the SC implementation, so sth weird worth reviewing is going on there